twentytwenty-blocks/functions.php

```diff
@@ -307,3 +307,16 @@ function twentytwentyblocks_register_styles() {
 	wp_enqueue_style( 'twentytwentyblocks-style', get_stylesheet_uri(), array(), $theme_version );
 }
 add_action( 'wp_enqueue_scripts', 'twentytwentyblocks_register_styles' );
+
+
+/**
+ * Register Block Patterns.
+ */
+
+register_pattern (
```

Comment:
Let's wrap this function call to be safe / avoid errors if the user is running an older version of Gutenberg.

```php
if ( function_exists( 'register_pattern' ) ) {
  register_pattern (
    ...
  )
}
```

Reply:
Pushed an update!